less/v2/journal.less

```
@@ -172,6 +172,132 @@
 
     p:last-of-type { margin-bottom: 0; }
   }
+
+  /* NPC Stat Blocks */
+  .statblock.npc {
+    --statblock-background: rgb(235 228 214 / .35);
+    --statblock-ability-header-1st: rgb(234 229 217);
+    --statblock-ability-header-2nd: rgb(215 217 208);
+    --statblock-ability-stat-1st: rgb(218 211 203);
+    --statblock-ability-stat-2nd: rgb(205 201 201);
+    --statblock-border: rgb(78 76 74);
+    --statblock-column-width: 400px;
+    --statblock-text-header: rgb(79 29 21);
+    --statblock-text-primary: rgb(31, 30, 30);
+    --statblock-text-secondary: rgba(73 72 73 / .75);
+
+    max-width: var(--statblock-column-width);
+    background: var(--statblock-background);
+    border: 3px double var(--statblock-border);
+    border-radius: 8px;
+    padding-block: 4px;
+    padding-inline: 8px;
+    box-shadow: 2px 2px 2px rgb(0 0 0 / .15);
+
+    h4.statblock-title, h5.statblock-actions-title {
+      border-block-end: 1px solid currentcolor;
+      font-variant: small-caps;
+    }
+    h4.statblock-title {
+      color: color-mix(in oklab, var(--statblock-text-header) 75%, transparent);
+      font-size: var(--font-size-18);
+    }
+    h5.statblock-actions-title {
+      margin-block-start: 8px;
+      color: var(--statblock-text-header);
+      font-size: var(--font-size-16);
+      font-weight: normal;
+    }
+    .statblock-tags {
+      color: var(--statblock-text-secondary);
+      font-style: italic;
+    }
+
+    .statblock-content {
+      display: column;
+      column-width: var(--statblock-column-width);
+    }
+    .statblock-header {
+      break-inside: avoid;
+      color: var(--statblock-text-header);
+
+      dl {
+        display: flex;
+        flex-wrap: wrap;
+
+        > div {
+          flex: 1 0 100%;
+          &.half-width { flex: 1 0 50%; }
```

Some future-proofing for v13 styles:

```suggestion
          &.half-width { flex: 1 0 50%; }
          dt {
            color: var(--statblock-text-header);
            text-shadow: none;
          }
```